DangoPlop/Assets/Scripts/PowerupMaster.cs

```diff
@@ -199,5 +211,21 @@ public class PowerupMaster : MonoBehaviour {
 		Vector3 originalPosition = playerObject.transform.position;
 	}
 
+	private void ResetBulletType(){
+		playerController.BulletType = "DefaultFire";
```

Comment:
in good object oriented design, classes shouldn't have direct access to other class's properties. To change their properties, they should call a public function to alter them. Make a public function called setBulletType in the player controller which you call here.